cime/scripts/lib/CIME/build.py

```diff
@@ -498,7 +519,7 @@ def _case_build_impl(caseroot, case, sharedlib_only, model_only, buildlist,
             os.environ["USE_KOKKOS"] = "TRUE"
```

Comment:
You might merge master into this branch and test again before merging this PR to next. This PR branches from master _before_ anything in master was building Kokkos on all machines. Now that PR #2896 is in master, all platforms are building Kokkos. On line 518 of this file (for example; there are other spots, too), only preqx_kokkos triggers Kokkos, whereas in master theta-l does, too, and theta-l builds in all nightlies, I believe.

Reply:
Will do.

Reply:
I just remembered one detail. Right now there are no coupled theta-l tests. That means Kokkos is being build only as part of the test `HOMME_P24.f19_g16_rx1.A`. So to test the full Kokkos capability (both standalone Homme build, which is tested currently, and the CIME build, which currently is not), build a case with `./xmlchange CAM_TARGET=theta-l`.

Reply:
Ok